<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>809</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Учет заявок на ремонт оргтехники</string>
  </property>
  <property name="windowIcon">
   <iconset theme="accessories-dictionary">
    <selectedon>../../../../../logo.png</selectedon>
   </iconset>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
  background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #5A7BD5, stop:1 #2C50BE);
}

QStackedWidget {
  background-color: #FFFFFF;
  border-radius: 10px;
  padding: 15px;
  /* Тень */
  box-shadow: 0px 4px 8px rgba(0,0,0,0.1);
}
QPushButton {
  background-color: #67BA80;
  border: none;
  border-radius: 8px;
  color: white;
  padding: 10px 20px;
  font-weight: bold;
  transition: background-color 0.3s ease;
}

QPushButton:hover {
  background-color: #57A970;
}

QPushButton:pressed {
  background-color: #4A8C5E;
}

/* Красные кнопки &quot;Выйти&quot;, &quot;Удалить&quot; */

QPushButton#backButton, QPushButton#btnLogout, QPushButton#deleteEmployeeButton, QPushButton#deleteStorageButton, QPushButton#deletePartnerButton {
  background-color: #E74C3C;
  border-radius: 8px;
  color: white;
}

QPushButton#backButton:hover, QPushButton#btnLogout:hover, QPushButton#deleteEmployeeButton:hover, QPushButton#deleteStorageButton:hover, QPushButton#deletePartnerButton:hover {
  background-color: #C0392B;
}

QLabel#titleLabel, QLabel#loginTitle {
  color: #FFFFFF;
  text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.4);
  font-family: &quot;Segoe UI&quot;;
  font-weight: 700;
}
QLineEdit, QComboBox, QSpinBox {
  border: 2px solid #ddd;
  border-radius: 6px;
  padding: 6px 10px;
  font-size: 14px;
}

QLineEdit:focus, QComboBox:focus, QSpinBox:focus {
  border-color: #67BA80;
  outline: none;
  background-color: #f9fff9;
}
QListWidget {
  background-color: #fff;
  border: 1px solid #ccc;
  border-radius: 8px;
  padding: 5px;
}

QListWidget::item {
  margin: 4px 0;
  padding: 12px;
  border-radius: 6px;
}

QListWidget::item:selected {
  background-color: #67BA80;
  color: white;
}

</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="Avtorisation">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>50</number>
         </property>
         <property name="topMargin">
          <number>50</number>
         </property>
         <property name="rightMargin">
          <number>50</number>
         </property>
         <property name="bottomMargin">
          <number>50</number>
         </property>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>12</number>
           </property>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <pointsize>18</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>АВТОРИЗАЦИЯ</string>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLineEdit" name="LoginField">
             <property name="placeholderText">
              <string>Логин</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="PasswordField">
             <property name="placeholderText">
              <string>Пароль</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="SignInButton">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(46, 194, 126);</string>
             </property>
             <property name="text">
              <string>Войти</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="ErrorField">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 0, 0);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="addusers">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Добавление пользователя</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QTableWidget" name="tableWidget"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit">
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>Telegram ID</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_2">
             <property name="placeholderText">
              <string>ФИО</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>Роль</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_3">
             <property name="placeholderText">
              <string>Логин</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_4">
             <property name="placeholderText">
              <string>Пароль</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="addusers_2">
           <item>
            <widget class="QPushButton" name="addus">
             <property name="text">
              <string>Добавить</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="apdateuser">
             <property name="text">
              <string>Изменить</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteuser">
             <property name="text">
              <string>Удалить</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>Назад</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="user">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="0">
          <widget class="QListWidget" name="listWidget"/>
         </item>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="navigation"/>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="importButton">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(46, 194, 126);</string>
             </property>
             <property name="text">
              <string>Импортировать заказы</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="lagPage">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(46, 194, 126);</string>
             </property>
             <property name="text">
              <string>Заказать поставку</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="adduser">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(46, 194, 126);</string>
             </property>
             <property name="text">
              <string>Добавить пользователя</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="sait">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(46, 194, 126);</string>
             </property>
             <property name="text">
              <string>На сайт компании</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="product">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QListWidget" name="listWidget_2">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>400</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="1">
              <widget class="QComboBox" name="productcomboBox">
               <property name="currentText">
                <string>Продукт</string>
               </property>
               <item>
                <property name="text">
                 <string>Продукт</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="countspinBox">
               <property name="specialValueText">
                <string>Кол-во продукта </string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="add">
             <property name="text">
              <string>Заказать товар</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="add_2">
           <property name="text">
            <string>Скачать заявку на поставку</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="add_3">
           <property name="text">
            <string>Назад</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="searchPage">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="2" column="1">
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>300</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="2">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>300</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <pointsize>18</pointsize>
               <weight>50</weight>
               <bold>false</bold>
               <underline>false</underline>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="text">
              <string>Назначить сборщика</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLineEdit" name="orderIdInput">
             <property name="placeholderText">
              <string>Номер заказа</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="statusComboBox">
             <item>
              <property name="text">
               <string>Сборщики</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>В ожидании</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Выполнено</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Отменено</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="search">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(46, 194, 126);</string>
             </property>
             <property name="text">
              <string>Назначить сборщика</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_8">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="AvtorButton_2">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(46, 194, 126);</string>
             </property>
             <property name="text">
              <string>Назад</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="nacklad">
             <property name="text">
              <string>Создать накладную</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <spacer name="verticalSpacer_7">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(36, 31, 49);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="pixmap">
         <pixmap>../media/am-Photoroom (1).png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>20</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>АМБАРЪ</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="AvtorButton">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(46, 194, 126);</string>
        </property>
        <property name="text">
         <string>Выход</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
